transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -209,12 +221,13 @@ private void registerService(BundleContext bundleContext) {
 
     private List<String> getSupportedMimeTypes() {
         SortedSet<MediaType> mediaTypes = MediaTypeRegistry.getDefaultRegistry().getTypes();
-        List<String> mimeTypes = new ArrayList<String>(mediaTypes.size());
+        List<String> mimeTypes = new ArrayList<>(mediaTypes.size());
 
         for (MediaType mediaType : mediaTypes) {
             String mimeType = mediaType.getType() + "/" + mediaType.getSubtype();
             mimeTypes.add(mimeType);
         }
+        mimeTypes.add("image/jp2");
```

Comment:
Not sure if you looked at it, but it seems like Tika 1.7 added support for jpeg2000 through their GDALParser. Not sure if it would be worth upgrading to 1.7 for that.